mail/postfix/src/opnsense/mvc/app/models/OPNsense/Postfix/Headerchecks.xml

```diff
@@ -0,0 +1,25 @@
+<model>
+    <mount>//OPNsense/postfix/headerchecks</mount>
+    <description>Postfix header_checks configuration</description>
+    <version>1.0.0</version>
+    <items>
+        <headerchecks>
+            <headercheck type="ArrayField">
+                <enabled type="BooleanField">
+                    <default>1</default>
+                    <Required>Y</Required>
+                </enabled>
+                <expression type="TextField">
+                    <Required>Y</Required>
+                </expression>
+                <filter type="OptionField">
+                    <Required>Y</Required>
+                    <OptionValues>
+                        <WHILE_DELIVERING>WHILE DELIVERING MAIL</WHILE_DELIVERING>
+                        <WHILE_RECEIVING>WHILE RECEIVING MAIL</WHILE_RECEIVING>
```

Comment:
I would not write that in uppercase

Reply:
Okay, I'll submit a commit changing this within the next few days. Do you mean the tag or the text or both?

Reply:
I only care for the display text - if possible it should be as translatable as possible.